.github/workflows/qat.yml

```diff
@@ -1,89 +1,75 @@
-name: QAT
+name: 'Checkly QAT'
```

Comment:
Let's leave this as `QAT`. The step name includes a "Checkly" reference, and if we ever introduced multiple QAT steps, we don't want to be tied to a specific provider.